### PR TITLE
Fix #1018

### LINKED ISSUE #1018: Bold headers not working in Chrome
Chrome:

image

Firefox:

Schermafbeelding 2021-03-22 om 17 05 42

### PR DESCRIPTION
**Pull request by lucifer1004**
This PR fixes #1018 by adding a CSS rule.

### CODE REVIEW

**Comment by pankgeorg** [comment]
Hey! Thanks for taking the time reviewing this! Quick question, can you provide some context on the source of the issue?
Can you test a CSS library like https://necolas.github.io/normalize.css/? Does it also fix it?
(I'm not saying we should include it per se, just see if it a normalization issue!)

**Comment by lucifer1004** [comment]
Source: [TEXT-BOLD Does Not Work Using HTML5](https://support.google.com/culturalinstitute/partners/answer/10460872)

>  It seems Chrome and Firefox (and other browsers) are rendering this in different ways. Firefox and Internet Explorer are making the bold text even bolder (font-weight: 900) when <strong> is used, while Chrome and Edge are not (font-weight: 700).

> It looks like if you explicitly set font-weight to 900 for the <strong> tags (actually "font-weight: bolder" might be best) then Chrome and Edge will work like Firefox and Internet Explorer.

**Comment by lucifer1004** [comment]
For the `bolder` option: [Meaning of relative weights](https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight#meaning_of_relative_weights)

**Comment by lucifer1004** [comment]
I had a look at the source code of `normalize.css` and it did exactly the [same](https://github.com/necolas/normalize.css/blob/fc091cce1534909334c1911709a39c22d406977b/normalize.css#L96).

File: frontend/editor.css [hunk 1]
```diff
@@ -177,6 +177,10 @@ pluto-output p:first-child {
     margin-block-start: 0px;
 }
 
+pluto-output strong {
```
Comment (pankgeorg) [inline frontend/editor.css:180]:
```suggestion
/* https://github.com/necolas/normalize.css/blob/fc091cce1534909334c1911709a39c22d406977b/normalize.css#L96 */
b,
strong {
```

**Comment by pankgeorg** [comment]
> I had a look at the source code of `normalize.css` and it did exactly the [same](https://github.com/necolas/normalize.css/blob/fc091cce1534909334c1911709a39c22d406977b/normalize.css#L96).

That is great! Thank you for searching this through!

**Pull request merged**